esp/esp/tagdict/__init__.py

```diff
@@ -0,0 +1,107 @@
+# Lists of all tags used anywhere in the codebase
+# Populated by hand, so don't be too surprised if something
+# is missing
+
+# Format: 'key': "some help text for admins"
+
+# Any tag used with Tag.getTag()
+all_global_tags = {
+    'teacherreg_custom_forms': "",
+    'allow_global_restypes': "Include global resource types in the options shown",
+    'splashinfo_choices': "",
+    'full_group_name': "",
+    'nearly_full_threshold': "",
+    'splashinfo_siblingdiscount': "",
+    'splashinfo_lunchsat': "",
+    'splashinfo_lunchsun': "",
+    'use_class_size_optimal': "",
+    'teacherreg_difficulty_choices': "",
+    'volunteer_tshirt_options': "",
+    'volunteer_tshirt_type_selection': "",
+    'volunteer_allow_comments': "",
+    'min_available_timeslots': "",
+    'availability_group_timeslots': "Group timeslots into contiguous blocks in the availability interface (enabled by default, set to 'False' to disable)",
+    'formstack_id': "",
+    'formstack_viewkey': "",
+    'group_phone_number': "",
+    'finaid_form_fields': "",
+    'onsite_classlist_min_refresh': "",
+    'studentschedule_show_empty_blocks': "",
+    'oktimes_collapse': "",
+    'qsd_display_date_author': "",
+    'current_theme_name': "",
+    'prev_theme_customization': "",
+    'current_theme_params': "",
+    'theme_template_control': "",
+    'current_theme_palette': "",
+    'request_student_phonenum': "",
+    'allow_change_grade_level': "",
+    'show_studentrep_application': "",
+    'show_student_tshirt_size_options': "",
+    'show_student_vegetarianism_options': "",
+    'show_student_graduation_years_not_grades': "",
+    'ask_student_about_transportation_to_program': "",
+    'student_medical_needs': "",
+    'require_school_field': "",
+    'teacherinfo_shirt_options': "",
+    'teacherinfo_shirt_type_selection': "",
+    'teacher_profile_hide_fields': "",
+    'student_grade_options': "",
+    'user_types': "",
+    'studentinfo_shirt_options': "",
+    'studentinfo_food_options': "",
+    'ask_about_duplicate_accounts': "",
+    'require_email_validation': "",
+}
+
+# Any tag used with Tag.getBooleanTag()
+all_boolean_tags = {
+    'increment_default_grade_levels': "Consider all students to be one grade level higher (useful for summer programs)",
```

Comment:
Some of these have docstrings here but not below.  It might be worth thinking about whether there's a good way to avoid duplication.